### PR TITLE
Implements the tests for committee members actions

### PR DESCRIPTION
**Pull request by romain-pierre**

### CODE REVIEW

File: app/config/config_test_mysql.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ imports:
     - { resource: config_test.yml }
 
 parameters:
-    env(DATABASE_HOST): "testdb"
+    env(DATABASE_HOST): "127.0.0.1"
```
Comment (hhamon) [inline app/config/config_test_mysql.yml]: I don't think this value shouls be changed!

File: tests/AppBundle/Committee/CommitteeUtilsTest.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+use Tests\AppBundle\MysqlWebTestCase;
+use Tests\AppBundle\TestHelperTrait;
+
+class CommitteeUtilsTest extends MysqlWebTestCase
+{
+    use TestHelperTrait;
+
+    public function testGetUuidsFromJson()
+    {
+        $this->assertEquals([], CommitteeUtils::getUuidsFromJson(''));
```
Comment (hhamon) [inline tests/AppBundle/Committee/CommitteeUtilsTest.php]: `assertSame`  is better to keep consistency.

File: tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+use AppBundle\DataFixtures\ORM\LoadAdherentData;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\PostAddress;
+use AppBundle\Membership\AdherentFactory;
+use Symfony\Component\Security\Core\Encoder\EncoderFactory;
+use Symfony\Component\Security\Core\Encoder\PlaintextPasswordEncoder;
+use Tests\AppBundle\MysqlWebTestCase;
+use Tests\AppBundle\TestHelperTrait;
+
+class AdherentCsvSerializerTest extends MysqlWebTestCase
```
Comment (hhamon) [inline tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php]: This class should be unit tested. We should not rely on database access.

File: src/AppBundle/Committee/CommitteeUtils.php [hunk 4]
```diff
@@ -42,13 +46,13 @@ public static function getUuidsFromJson(string $json): array
...
         }
 
         foreach ($uuids as $uuid) {
             foreach ($adherents as $adherent) {
                 if (!$adherent instanceof Adherent) {
-                    throw new \BadMethodCallException('This method requires a collection of Adherent entities');
+                    throw new AdherentCollectionException();
                 }
 
                 if ((string) $adherent->getUuid() === (string) $uuid) {
```
Comment (hhamon) [inline src/AppBundle/Committee/CommitteeUtils.php]: Why not using the `equals()` method of the `UUID` class?

File: tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php [hunk 5]
```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Tests\AppBundle\Committee\Serializer;
+
+use AppBundle\Committee\Serializer\AdherentCsvSerializer;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\PostAddress;
+use AppBundle\Membership\AdherentFactory;
+use PHPUnit\Framework\TestCase;
```
Comment (hhamon) [inline tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php]: We're not using PHPUnit 6...

File: tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+use Symfony\Component\Security\Core\Encoder\EncoderFactory;
+use Symfony\Component\Security\Core\Encoder\PlaintextPasswordEncoder;
+
+class AdherentCsvSerializerTest extends TestCase
```
Comment (hhamon) [inline tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php]: Must be PHPUnit_Framework_TestCase

File: tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php (hunk 5 above)
Comment (romain-pierre) [inline tests/AppBundle/Committee/Serializer/AdherentCsvSerializerTest.php]: This is valid also.

File: src/AppBundle/Committee/CommitteeUtils.php (hunk 4 above)
Comment (romain-pierre) [inline src/AppBundle/Committee/CommitteeUtils.php]: Because this could be an Uuid object OR a simple string.

**Pull request merged**